libr/core/cbin.c

```diff
@@ -1873,6 +1873,18 @@ static bool isAnExport(RBinSymbol *s) {
 	return (s->bind && !strcmp (s->bind, R_BIN_BIND_GLOBAL_STR));
 }
 
+static void select_flag_space(RCore *core, RBinSymbol *symbol) {
+	if (!strncmp (symbol->name, "imp.", 4)) {
```

Comment:
why is "imp." inlined and not using a define? what about sym.imp.?

Reply:
i think that we can check if a symbol is an import by checking the irght propert¡y instead of asuming the name contains all the information

Reply:
i think symbol->type should be "import" or something likt that

Reply:
this would require a lot of changes in all plugins... I'm not sure that's something worth at this point.